### PR TITLE
Docs: Fixes for 11.1 visualizations docs

### PR DESCRIPTION
**Pull request by imatwawana**
Fixes some wording in table docs
Fixes some formatting in stat docs
Adds an availability note in xy chart docs

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 1]
```diff
@@ -120,17 +120,17 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
 
 ### Auto
 
-The Auto cell type will automatically display values with sensible defaults applied.
+The **Auto** cell type automatically displays values, with sensible defaults applied.
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field you've set a field override.
```
Comment (adela-almasan) [inline docs/sources/panels-visualizations/visualizations/table/index.md:125]: > or the matched field you've set a field override.

This sounds odd to me.. 🤔

Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/table/index.md:125]: Whoops I missed a word. Updating.

Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:125]:
```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 2]
```diff
@@ -120,17 +120,17 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
...
 
 ### Color text
 
 If thresholds are set, then the field text is displayed in the appropriate threshold color.
 
 {{< figure src="/static/img/docs/tables/color-text.png" max-width="500px" caption="Color text" class="docs-image--no-shadow" >}}
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field you've set a field override.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:133]:
```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 {{< figure src="/static/img/docs/tables/colored-rows.png" max-width="500px" alt="Colored row background" class="docs-image--no-shadow" >}}
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field you've set a field override.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:145]:
```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched if field you've set a field override.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 4]
```diff
@@ -120,17 +120,17 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
 
 ### Auto
 
-The Auto cell type will automatically display values with sensible defaults applied.
+The **Auto** cell type automatically displays values, with sensible defaults applied.
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
```
Comment (adela-almasan) [inline docs/sources/panels-visualizations/visualizations/table/index.md:125]: maybe a comma? no strong feelings either way.

```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field, if you've set a field override.
```

**Review by adela-almasan — APPROVED** [review]
⭐

File: docs/sources/panels-visualizations/visualizations/table/index.md (hunk 4 above)
Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/table/index.md:125]: Works for me!

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 {{< figure src="/static/img/docs/tables/colored-rows.png" max-width="500px" alt="Colored row background" class="docs-image--no-shadow" >}}
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched if field you've set a field override.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:145]:
```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched, if field you've set a field override.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 6]
```diff
@@ -120,17 +120,17 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
...
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
 
 ### Color text
 
 If thresholds are set, then the field text is displayed in the appropriate threshold color.
 
 {{< figure src="/static/img/docs/tables/color-text.png" max-width="500px" caption="Color text" class="docs-image--no-shadow" >}}
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:133]:
```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field, if you've set a field override.
```

**Pull request merged**